lib/message.rb

```diff
@@ -0,0 +1,11 @@
+class Message
+  def initialize(statement, morse)
+    @statment = statement
+    @morse = morse
+
+  end
+
+  def translation
+    @morse.translate("There are 3 ships")
```

Comment:
This shouldn't just translate "There are 3 ships". Instead, you should use the @statement that was passed into initialize